backend-api/app/routes/consumers.py

```diff
@@ -6,24 +6,32 @@
 
 router = APIRouter()
 
-consumer_dict = {"ethereum_address": "somewhareOnTheBlockchain"}
+consumer_dict = {"eth_address": "somewhareOnTheBlockchain"}
 
 
 @router.post("/", status_code=status.HTTP_201_CREATED)
 async def create_consumer(consumer: ConsumerCreate):
+    # Create database instance, retrieve ID
+
     return ConsumerRead(id=random.randint(1, 1000), **consumer_dict)
 
 
-@router.get("/me")
-async def read_consumer():
+@router.get("/{eth_address}}", status_code=status.HTTP_200_OK)
```

Comment:
The 200 OK is the default, so I didn't include it. I'm okay with being more explicit.

Reply:
I kept them all declared them all explicitly for now 👍